.github/workflows/codeql.yml

```diff
@@ -0,0 +1,58 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "v1", "cloud-*", "master", "release/*" ]
+  pull_request:
+    branches: [ "v1", "cloud-*", "master", "release/*" ]
+  schedule:
+    - cron: '36 17 * * 0'
+  workflow_call:
+    inputs:
+      ref:
+        required: true
+        type: string
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
```

Comment:
We can simplify this since there is only one language.  Use `Analyze Go` here, and use `ubuntu-latest` on the next line.  Similarly, we can remove the `strategy` portion and use the values directly in the steps.

Reply:
Ah, good catch! I'll update that to simplify the scan.

Reply:
I'm going to merge this as-is for now to unblock work, we can follow up with a refactor.